core/kernel_init.c

```diff
@@ -66,14 +68,9 @@ static void *idle_thread(void *arg)
     (void) arg;
 
     while (1) {
-        if (lpm_prevent_sleep) {
-            lpm_set(LPM_IDLE);
-        }
-        else {
-            lpm_set(LPM_IDLE);
-            /* lpm_set(LPM_SLEEP); */
-            /* lpm_set(LPM_POWERDOWN); */
-        }
+#ifdef FEATURE_PERIPH_PM
+        pm_set_lowest();
+#endif
```

Comment:
in case PM is not present, wouldn't 
```c
#else
    while (1) {}
#endif
```
be more precise, so we prevent the scheduler from looping?